ambry-account/src/test/java/com/github/ambry/account/MySqlAccountServiceTest.java

```diff
@@ -0,0 +1,336 @@
+package com.github.ambry.account;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.config.MySqlAccountServiceConfig;
+import com.github.ambry.config.VerifiableProperties;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import org.junit.Test;
+
+import static com.github.ambry.config.MySqlAccountServiceConfig.*;
+import static com.github.ambry.utils.TestUtils.*;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+
+public class MySqlAccountServiceTest {
+
+  AccountService mySqlAccountService;
+  Properties mySqlConfigProps = new Properties();
+
+  public MySqlAccountServiceTest() {
+    resetConfig();
+  }
+
+  private void resetConfig() {
```

Comment:
Also, instead of calling this at the end of each method, you could add the @Before tag to have it called before each test automatically (probably need to make it public).